lib/rucio/db/sqla/models.py

```diff
@@ -1184,6 +1192,23 @@ class Token(BASE, ModelBase):
                    Index('TOKENS_ACCOUNT_EXPIRED_AT_IDX', 'account', 'expired_at'))
 
 
+class OAuthRequest(BASE, ModelBase):
+    """Represents the authentication session parameters of OAuth 2.0 requests"""
+    __tablename__ = 'oauth_requests'
+    account = Column(InternalAccountString(25))
```

Comment:
add foreignkeyconstraint

Reply:
I had this originally in place, but due to webui not having the possibility to ask the user which account to use, I had to drop it. I discussed this already with Martin. I would need to implement a Rucio login page with options (which is of course possible as well, we just need to discuss it a bit).

Reply:
Correct. If more accounts have the same identity, one will be chosen (default/oldest).
I have started implementing the login page where optionally entering account name will be possible. I will keep this new dev off this pull request we have this merged.